server/apps/archive/archive_spike.py

```diff
@@ -69,9 +69,21 @@ class ArchiveSpikeService(BaseService):
 
     def on_update(self, updates, original):
         updates[ITEM_OPERATION] = ITEM_SPIKE
+        self._validate_item(original)
         self._validate_take(original)
         self._update_rewrite(original)
 
+    def _validate_item(self, original):
+        '''
```

Comment:
Technically speaking it can be any string literal, but yes, the convention is to use triple double quotes indeed.